src/coreComponents/dataRepository/Group.cpp

```diff
@@ -632,14 +632,42 @@ Group const & Group::getBaseGroupByPath( string const & path ) const
     }
     else
     {
-      currentGroup = &currentGroup->getGroup( curGroupName );
+      if( hardQuery || currentGroup->hasGroup( curGroupName ) )
+      {
+        currentGroup = &currentGroup->getGroup( curGroupName );
+      }
+      else
+      {
+        break;
+      }
     }
   }
   while( currentPosition != string::npos );
 
   return *currentGroup;
 }
 
+bool Group::hasGroupByPath( string const & path ) const
```

Comment:
Is this called anywhere? My search doesn't return anything???

Reply:
No I can remove it. It was used in a previous version.